grapher/stackedCharts/MarimekkoChart.tsx

```diff
@@ -211,13 +212,22 @@ function MarimekkoBarsForOneEntity(props: MarimekkoBarsProps): JSX.Element {
 
     let content = undefined
     if (bars.length) {
-        const allButLast = bars.slice(0, -1)
-        const last = bars[bars.length - 1]
+        const lastNonZeroBarIndex =
+            bars.length -
+            bars.reverse().findIndex((bar) => bar.yPoint.value > 0) -
+            1
```

Comment:
nit: don't have to but can use [`findLastIndex`](https://lodash.com/docs/4.17.15#findLastIndex) from lodash (just needs to be exposed in `Util.ts`)

Reply:
Nice!